packages/loot-core/src/shared/rules.ts

```diff
@@ -57,6 +58,8 @@ export function mapField(field, opts) {
       return 'amount (inflow)';
     case 'amount-outflow':
       return 'amount (outflow)';
+    case 'is_transfer':
+      return 'Transfer';
```

Comment:
Since all of the other field names are lower case, this one should be too:

```suggestion
      return 'transfer';
```